<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab7</class>
 <widget class="QWidget" name="Tab7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1513</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>0</y>
     <width>511</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QToolButton" name="toolButton_52">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>50</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>选项</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_53">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>50</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_54">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>50</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_55">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>50</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
